### DISCUSSION TITLE
Restarting an old project

### DISCUSSION THREAD

**Discussion by jonphipps** [Laravel-Backpack/community-forum#878] [Q&A (Help)] [answered]
I have a backpack/crud 3.3, laravel 5.5, PHP 7.1 project that I need to update. It was actually already updated from Symfony 1 and there are parts of it that are just wrappers around that 2008 code -- GitHub says there are 1,700 PHP files in the repo. I'm not actually thinking of just updating, but intend to start from scratch.

My question is... how likely is it that I will be able to pull any of my working 3.3 crud code into 6.x? 

I know it's a lazy question, but at this point I'm unfamiliar with both 3.3 and 6.x, and my first impression is that they look pretty different. For sure I'm not going to step through each version upgrade.

**Comment by jcastroa87** [comment, marked as answer]
Hello @jonphipps 

Jump from 3 to 6 will be a very big jump; you will notice there are many breaking changes, and there are plenty of new fields and features, I don't know about your project, but if early you use the image field to handle photos, now you want to try dropzone, as an example. If you want to reuse some code, could be of basic fields like text, email, etc. But I think in other kinds of fields you want to try new fields that will make everything easier.

You will love the new version of Backpacks and will be surprised by how fast you can develop any new project.

Cheers.
